model/convert/fixtures/fixture.go

```diff
@@ -173,7 +173,7 @@ var epochSetupFixtureJSON = `
                   {
                     "name": "id",
                     "value": {
-                      "type": "String",
+                      "type": "Name",
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig